alpine/entrypoint/post_start.sh

```diff
@@ -5,47 +5,13 @@ do
    [[ "${LOGLINE}" == *"Binding bundle: [landing-page"* ]] && pkill -P $$ tail
 done
 
-if [ -n "$INSTALL_FEATURES" ]; then
-  if [[ $INSTALL_FEATURES == *";"* ]]; then
-  _featureCount=$[$(echo $INSTALL_FEATURES | grep -o ";" | wc -l) + 1]
-  else
-    _featureCount=1
-  fi
+$APP_HOME/bin/client waitForReady -r 12 -d 10
+$APP_HOME/bin/client admin:configure /opt/ddfConfig.json -r 12 -d 10
```

Comment:
This is definitely subjective, but I think I'd rather avoid another environment variable. If they're running this container on a distribution without the ConfigureCommand, it will just say "command not found" and move on. If the json config file doesn't exist, nothing will happen. So this call is very harmless if something is missing.